dashboard/db/migrate/20170120193705_change_id_column_type.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeIdColumnType < ActiveRecord::Migration[5.0]
+  def up
+    change_column :activities, :id, :bigint
+  end
+
+  def down
+    change_column :activities, :id, :integer
```

Comment:
I checked in a change that comments this out; I'm on the fence, though, happy to put it back if anyone feels strongly.